operators/katib-controller/reactive/katib_controller.py

```diff
@@ -0,0 +1,394 @@
+                                "resources": ["customresourcedefinitions"],
+                                "verbs": ["create", "get"],
+                            },
+                            {
+                                "apiGroups": ["admissionregistration.k8s.io"],
+                                "resources": [
+                                    "validatingwebhookconfigurations",
+                                    "mutatingwebhookconfigurations",
+                                ],
+                                "verbs": ["*"],
+                            },
+                            {
+                                "apiGroups": ["kubeflow.org"],
+                                "resources": [
+                                    "experiments",
+                                    "experiments/status",
+                                    "experiments/finalizers",
+                                    "trials",
+                                    "trials/status",
+                                    "trials/finalizers",
+                                    "suggestions",
+                                    "suggestions/status",
+                                    "suggestions/finalizers",
+                                ],
+                                "verbs": ["*"],
+                            },
+                            {
+                                "apiGroups": ["kubeflow.org"],
+                                "resources": [
+                                    "tfjobs",
+                                    "pytorchjobs",
+                                    "mpijobs",
+                                ],
+                                "verbs": ["*"],
+                            },
+                            {
+                                "apiGroups": ["tekton.dev"],
+                                "resources": [
+                                    "pipelineruns",
+                                    "taskruns",
+                                ],
+                                "verbs": ["*"],
+                            },
+                            {
+                                "apiGroups": ["rbac.authorization.k8s.io"],
+                                "resources": [
+                                    "roles",
+                                    "rolebindings",
+                                ],
+                                "verbs": ["*"],
+                            },
+                        ]
+                    }
+                ]
+            },
+            "containers": [
+                {
+                    "name": "katib-controller",
+                    "command": ["./katib-controller"],
+                    "args": [
+                        "-webhook-port",
+                        str(config["webhook-port"]),
+                        "-logtostderr",
+                        "-v=4",
```

Comment:
Those were there to ensure that we got useful logging from the pod. They were taken from `./katib-controller --help`, but I've removed them since they should be unnecessary.